c++/src/Vector.cc

```diff
@@ -436,4 +438,8 @@ namespace orc {
           + static_cast<uint64_t>(
               (data.capacity() + nanoseconds.capacity()) * sizeof(int64_t));
   }
+
+  int64_t TimestampVectorBatch::getSecondInWriterTZ(uint64_t rowId) {
+    return data[rowId] + writerTimezone->getVariant(data[rowId]).gmtOffset;
```

Comment:
Can we wrap the logic `ts + timezone->getVariant(ts).gmtOffset` as a function (e.g. `Timezone::getLocalSecs(long utcSecs)`) in Timezone? Looks like we frequently use it.

Reply:
Good suggestion!